composer.json

```diff
@@ -14,7 +14,9 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.0",
+        "ext-memcache": "*",
+        "ext-json": "*"
```

Comment:
This is not needed as `json_encode` and `json_decode` are part of the core php.

Reply:
The json extension is part of core php, however it can be disabled. I suggest keeping the requirement

Reply:
Yeah, Removed "ext-json" from the list.

Reply:
Added back the ext-json require. Anyways it doesn't make any difference.